Packs/GetLicenseID/Scripts/GetLicenseID/GetLicenseID.py

```diff
@@ -12,5 +12,5 @@ def get_license_id():
     return human_readable, {'Demisto License': license_id}
 
 
-if __name__ == '__main__':
+if __name__ in ('__main__', 'builtins'):
```

Comment:
```suggestion
if __name__ in ('__main__', '__builtin__', 'builtins'):
```

Reply:
But this uses a docker with python 3 and `__builtin__` is from python 2

Reply:
user can override the docker image